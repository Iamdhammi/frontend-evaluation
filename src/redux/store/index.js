import { createStore, applyMiddleware, compose } from "redux";
import thunk from "redux-thunk";
import rootReducer from "../reducers";

const initialState = {};

const middleware = [thunk];

// create Store
const store = createStore(
    rootReducer, // all combined reducers
    initialState, // DUHH initaial state at craete store level
    compose(
        applyMiddleware(...middleware)
    )
);

export default store;