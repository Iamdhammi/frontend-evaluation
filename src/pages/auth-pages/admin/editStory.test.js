import React from 'react';
import { render, cleanup, act } from '@testing-library/react';
import { Provider } from 'react-redux';
import store from '../../../redux/store';
import EditStory from './editStory';
import { Router, Route } from "react-router-dom";
import history from '../../../history'

afterEach(cleanup);

it("loads edit story", () => {
    const { asFragment } = render(
        <Provider store={store}>
            <Router history={history}>
                <Route path="/admin/stories/:id">
                    <EditStory />
                </Route>
            </Router>
        </Provider>
    );
    expect(asFragment()).toMatchSnapshot();
});
