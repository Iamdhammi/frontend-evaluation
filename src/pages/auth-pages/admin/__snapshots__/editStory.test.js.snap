// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`loads edit story 1`] = `<DocumentFragment />`;
